clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -35,8 +36,9 @@
                                                                      INT32,
                                                                      "A user-supplied integer value that will be passed back with the response"),
                                                            new Field("client_id",
```

Comment:
@jjkoshy Thats correct. Trunk changed from using the legacy requests and responses to the ones in the clients library between the 0.9 release and this report. I will need to make a separate patch for the 0.9.0 library but want to fix trunk first. More about the difference in failure can be read on the jira [here](https://issues.apache.org/jira/browse/KAFKA-3088?focusedCommentId=15130571&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15130571)